pkgs/development/compilers/spirv-llvm-translator/default.nix

```diff
@@ -96,7 +100,8 @@ stdenv.mkDerivation {
     "-DLLVM_SPIRV_BUILD_EXTERNAL=YES"
     # RPATH of binary /nix/store/.../bin/llvm-spirv contains a forbidden reference to /build/
     "-DCMAKE_SKIP_BUILD_RPATH=ON"
-  ] ++ lib.optional (llvmMajor != "11") "-DLLVM_EXTERNAL_SPIRV_HEADERS_SOURCE_DIR=${spirv-headers.src}";
+  ] ++ lib.optional (llvmMajor != "11") "-DLLVM_EXTERNAL_SPIRV_HEADERS_SOURCE_DIR=${spirv-headers.src}"
+    ++ lib.optional (llvmMajor == "19") "-DBASE_LLVM_VERSION=${lib.versions.majorMinor llvm.version}.0";
```

Comment:
Should this be `>`, or is there a reason to expect it will change for 20?

Reply:
Maybe when we add it to the versions set then we'll change this if needed.

Reply:
Is the failure obvious or is it something people might forget to do?

Reply:
It'll be obvious in the configure phase. The CMake files runs a version check which would cause the entire build to fail.